suite-native/device/src/deviceThunks.ts

```diff
@@ -0,0 +1,41 @@
+import { pipe, A } from '@mobily/ts-belt';
+
+import { createThunk } from '@suite-common/redux-utils';
+import {
+    selectDevices,
+    selectAccounts,
+    accountsActions,
+    selectDiscovery,
+    discoveryActions,
+    PORTFOLIO_TRACKER_DEVICE_STATE,
+} from '@suite-common/wallet-core';
+import { Account } from '@suite-common/wallet-types';
+
+const actionPrefix = '@suite-native/device';
+
+export const wipeDisconnectedDevicesDataThunk = createThunk(
+    `${actionPrefix}/wipeDisconnectedDevicesDataThunk`,
+    (_, { getState, dispatch }) => {
+        // TODO: When we enable remember mode, also `disconnected remembered` (device.remember === true) devices should stay untouched.
+        const connectedDevices = selectDevices(getState());
+        const connectedDeviceStates = [
+            ...connectedDevices.map(d => d.id),
+            PORTFOLIO_TRACKER_DEVICE_STATE,
+        ];
+
+        pipe(
```

Comment:
I thinking if we may need this to some other place in future? If no, it's okay to keep it here otherwise these two pipes could be selectors.

Reply:
You are right, this could be reused later. I created selectors for these.